runtime/mock/src/contracts_config.rs

```diff
@@ -135,10 +137,138 @@ impl FeeCalculator for FixedGasPrice {
     }
 }
 
+impl PortalReadApi<BlockNumber> for Runtime {
```

Comment:
Also not needed with new stuff, willl omit any more comments from the previous precompile PR from here on